lhotse/features/kaldifeat.py

```diff
@@ -193,6 +193,10 @@ def __init__(self, config: Optional[KaldifeatFbankConfig] = None) -> None:
     def feature_dim(self, sampling_rate: int) -> int:
         return self.config.mel_opts.num_bins
 
+    @property
+    def padding_value(self) -> float:
+        return -1000.0 if self.config.use_log_fbank else EPSILON
```

Comment:
Should -1000 be replaced with 
https://github.com/lhotse-speech/lhotse/blob/14b51f6e08166725c0e6b2ac29a210f61af4e684/lhotse/utils.py#L46

Reply:
Yes, ideally it should, but I was basing this on the default value used in the `FeatureMixer`.